libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -66,7 +66,11 @@ void copyMissingTags(set<CallableDeclaration const*> const& _baseFunctions, Stru
 				size_t docParaNameEndPos = content.content.find_first_of(" \t");
 				string const docParameterName = content.content.substr(0, docParaNameEndPos);
 
-				if (docParameterName != _declaration->returnParameters().at(n)->name())
+				if (
+					_declaration->returnParameterList() &&
+					_declaration->returnParameters().size() > n &&
```

Comment:
Won't this https://github.com/ethereum/solidity/blob/83fc7d3f7de5d8c0ce184dab4a79e2248b930c89/libsolidity/analysis/DocStringTagParser.cpp#L181 be triggered before? Can we move this into an assertion?

Reply:
No, this won't trigger because what we have is not a `FunctionDefinition` but a `ModifierDefinition`